tests/10apidoc/30room-create.pl

```diff
@@ -26,13 +26,9 @@
          assert_json_nonempty_string( $body->{room_alias} );
 
          Future->done(1);
-      });
-   },
-
-   check => sub {
-      my ( $user ) = @_;
-
-      matrix_initialsync( $user )->then( sub {
+      })->then( sub {
+         matrix_initialsync( $user );
```

Comment:
Rather that this, use the same `limit` trick as before. At the top of the file put something like

    # An incrementing parameter for initialSync to defeat the caching mechanism and ensure fresh results every time
    my $initial_sync_limit = 1;

Then change every occurance of `matrix_initialsync( $user )` to

    matrix_initialsync( $user, limit => $initial_sync_limit++ );